src/FakeItEasy/Configuration/ArgumentCollection.cs

```diff
@@ -11,7 +11,9 @@ namespace FakeItEasy.Configuration
     /// <summary>
     ///   A collection of method arguments.
     /// </summary>
+#if FEATURE_BINARY_SERIALIZATION
     [Serializable]
+#endif

```

Comment:
The attribute is coming back in future release though https://github.com/dotnet/corefx/commit/db70c8e2ba100b2f0f8fa1ace82dd1988200b269.